layers/+completion/ivy/config.el

```diff
@@ -13,7 +13,10 @@
 ;; Variables
 
 (defvar spacemacs--counsel-commands
-  '(("rg" . "rg --smart-case --no-heading --color never %s %S .")
+  '(;; -n forces line numbers (disabled by default on windows)
+    ;; no --vimgrep because it adds column numbers that wgrep can't handle
+    ;; (see https://github.com/syl20bnr/spacemacs/pull/8065)
+    ("rg" . "rg --smart-case --no-heading --color never -n %s %S .")
```

Comment:
I prefer long-form flags when not typing them at a terminal myself: `--line-number`